hangman.js

```diff
@@ -0,0 +1,139 @@
+var inquirer = require("inquirer");
+var Word = require("./word.js");
+
+// Prearranged words for the hangman program
+var words = ['PROGRAM', 'LANGUAGE', 'STRING', 'JAVASCRIPT', 'PYTHON'];
+
+// Array to store word Objects that are created based on the words above
+var wordArray = [];
+
+
+// Function to initalize wordArray with word Objects
+function initWords() {
+	for (var i=0; i<words.length; i++) {
+		var newWord = new Word(words[i]);
+		wordArray.push(newWord);
+	}
```

Comment:
You can just use a higher order function called map like so:
```javascript
return words.map(function(word){
  return new Word(word)
})
```
this will return a new array with each word having been passed through the Word constructor.